utils/build.config

```diff
@@ -4,7 +4,7 @@ component=daos
 [commit_versions]
 ARGOBOTS = 31703b1e97b4aa9a8b2bfc18415ce5146e853c2a
 CART = 24c97eab7b97df49e8e26a4618157a806e92cbad
-PMDK = f9c64502747534a958326b72d694cf952c74c0c3
+PMDK = 1.5.1
```

Comment:
Using version numbers now instead of hash??

Reply:
You can specify a tag or a hash.  In this case, it’s a tagged release.  It corresponds to a hash